nvbios/nvbios.c

```diff
@@ -1704,6 +1705,7 @@ int main(int argc, char **argv) {
 			entry_length = bios->data[start+2];
 			entry_count = bios->data[start+3];	// XXX: NFI what the entries are for
 			is_pwm_based = bios->data[start+4] & 1;
+			is_header_based = bios->data[start+4] & 0x2;
```

Comment:
ok, new flag, cool. What I don't like is the name because it seems to imply that it is sufficient information to know if we need to use the header or the entries, which is wrong because PWM tables will always use the header, no matter what this bit is.

So, how about gpio_header_mode or prefer_header?

How about prefer_header_src?